presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -427,6 +430,10 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
         boolean external;
         String externalLocation = getExternalLocation(tableMetadata.getProperties());
         if (externalLocation != null) {
+            if (!createNonManagedTableEnabled) {
+                throw new PrestoException(NOT_SUPPORTED, "Cannot create non-managed Hive table");
```

Comment:
It is supported, but it is disabled.  Try to find a better error code if possible.